packages/cozy-stack-client/src/TriggerCollection.js

```diff
@@ -27,6 +27,22 @@ export const isForAccount = (triggerAttrs, accountId) => {
   return triggerAttrs.message.account == accountId
 }
 
+const buildFindUrl = (worker, type) => {
+  let url = '/jobs/triggers?'
+  if (worker) {
+    const suffix = type ? '&' : ''
+    url += `Worker=${worker}${suffix}`
+  }
+  if (type) {
+    if (Array.isArray(type.$in)) {
+      url += `Type=${type.$in.join(',')}`
+    } else {
+      url += `Type=${type}`
+    }
+  }
```

Comment:
Could the URLSearchParams API  be used instead of building the params manually ?

https://developer.mozilla.org/fr/docs/Web/API/URLSearchParams

```
sp = new URLSearchParams()
URLSearchParams {  }

sp.append('Worker', 'toto')
sp.toString()
"Worker=toto"
sp.append('Type', '@cron')
sp.toString()
"Worker=toto&Type=%40cron"
```

@nono Will the "%40" be correctly parsed by the stack ? I see that in the doc, the "@" is not url-encoded. cf https://github.com/cozy/cozy-stack/pull/2884/files